.ci/scripts/utils.sh

```diff
@@ -124,7 +124,7 @@ build_executorch_runner() {
   if [[ $1 == "buck2" ]]; then
     build_executorch_runner_buck2
```

Comment:
for buck we are ignoring and using whatever's default?

Reply:
I don't think it's necessary to do both a debug and a release build with buck2 specifically; whether the code works with optimizing compiler flags should be orthogonal to choice of build system.

Reply:
yes. I think that whether the code works with optimized/debug build flags is orthogonal to the choice of build system. (The ExecuTorch .buckconfig is configured to produce debug builds by default.)